README.md

```diff
@@ -7,3 +7,6 @@ sanity checks of platform functionality.
 
 ## x-kiosk
 Runs the chromium browser inside a container and displays a website.
+
+## k3s
+Runs a minimal kubernetes installation using Rancher's k3s project.
```

Comment:
You probably should add something like:
***NOTE*** This requires a lot of memory. A raspberrypi3 is barely able to run this.

Reply:
added a note, I'm tippy toeing around the wording because I'm trying to stay concise and also prevent any unnecessary changes in the future (in the case that the memory constraints change)